docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
Since there are fields that can't be updated (e.g., PortalIP), we probably also need to support `reload`, to delete and recreate an object with a new configuration.

We probably should think about creation via update and update via creation (in the case that the object exists but differs).